### PR TITLE
update documentation about LOCAL ssd

### PR DESCRIPTION
**Pull request by lbergelson**

### CODE REVIEW

File: docs/RuntimeAttributes.md [hunk 1]
```diff
@@ -193,15 +193,15 @@ This attribute specifies volumes that will be mounted to the VM for your job. Th
...
+2. Disk size in GB (rounded to the next 375 GB for LOCAL)
 3. Disk type.  One of: "LOCAL", "SSD", or "HDD" ([documentation](https://cloud.google.com/compute/docs/disks/#overview))
 
 All tasks launched on Google Cloud *must* have a `local-disk`.  If one is not specified in the runtime section of the task, then a default of `local-disk 10 SSD` will be used.  The `local-disk` will be mounted to `/cromwell_root`.
 
 For the AWS Batch backend, the disk volume is managed by AWS EBS with autoscaling capabilities.  As such, the Disk size and disk type will be ignored. If provided, the mount point will be verified at runtime.
 
 
-The Disk type must be one of "LOCAL", "SSD", or "HDD". When set to "LOCAL", the size of the drive is automatically provisioned by Google so any size specified in WDL will be ignored. All disks are set to auto-delete after the job completes.
+The Disk type must be one of "LOCAL", "SSD", or "HDD". When set to "LOCAL", the size of the drive is constrained to 375 GB intervals so intermediate values will be rounded up to the next 375 GB.
```
Comment (dinvlad) [inline docs/RuntimeAttributes.md]: `All disks are set to auto-delete after the job completes.` should be kept I think

Comment (lbergelson) [reply docs/RuntimeAttributes.md]: Oh, you are very right.  This is why I should never edit in the github editor...

**Comment by lbergelson** [comment]
This is failing travis but my guess is that that's not my fault?

**Comment by cjllanwarne** [comment]
@lbergelson right, these look like transient failures so I'l give the tests a quick nudge...

**Pull request merged**